modules/material/themes/material/default/selectidp-links.twig

```diff
@@ -78,36 +78,9 @@
         </div>
       {% endfor %}
 
-      {% for idp in disabled_idps %}
-        <div
-          class="mdl-card mdl-shadow--2dp disabled row-aware"
-          title="{{ '{selectidp-links:disabled}'|trans({'%idpName%': idp|entityDisplayName}) }}"
-          onclick="clickedAnyway('{{ idp|entityDisplayName }}')"
-        >
-          <div class="mdl-card__media white-bg fixed-height" layout-children="row" child-spacing="center">
-            <div class="logo-container fill-parent">
-              <div class="image-wrapper">
-                <img
-                  class="logo"
-                  id="{{ idp.entityid|e }}"
-                  src="{{ idp.logoURL|default('default-logo.png') }}"
-                >
-              </div>
-              <span class="mdl-color-text--grey-600 logo-caption">
-                {{ idp.logoCaption|e|default('<br>') }}
-              </span>
-            </div>
-          </div>
-        </div>
-      {% endfor %}
     </form>
   </main>
 
-  <script>
-    ga('send', 'event', 'hub', 'choices', 'enabled', {{ enabled_idps|length }});
```

Comment:
Do we still want to send this event off? (with idp_list)

Reply:
Seems like pretty useless information to me. GA makes more sense for discovering user behavior. But it's all useless because nobody has ever used the data we collect anyway.